rust/shalom_dart_codegen/templates/operation.dart.jinja

```diff
@@ -143,4 +175,24 @@ class {{ root_type_name }}{
     }
 {% endfor %}
 
-// ------------ END OBJECT DEFINITIONS -------------
+
+
+// ------------ Enum DEFINITIONS -------------
+{% for full_name, selection in operation.context.type_defs | items  if selection.kind == "Enum" %}
+     enum {{ full_name }}  {
+         {% for name, value in selection.concrete_type.values | items %}  
+              {{ value.value }},
```

Comment:
should be name although it doesn't really matter since the current spec says that they should be equal (name and value). anyways use name.